source/Irrlicht/COpenGLExtensionHandler.cpp

```diff
@@ -1326,6 +1328,16 @@ void COpenGLExtensionHandler::loadFunctions()
     pGlDrawElementsIndirect = (PFNGLDRAWELEMENTSINDIRECTPROC) IRR_OGL_LOAD_EXTENSION("glDrawElementsIndirect");
     pGlMultiDrawArraysIndirect = (PFNGLMULTIDRAWARRAYSINDIRECTPROC) IRR_OGL_LOAD_EXTENSION("glMultiDrawArraysIndirect");
     pGlMultiDrawElementsIndirect = (PFNGLMULTIDRAWELEMENTSINDIRECTPROC) IRR_OGL_LOAD_EXTENSION("glMultiDrawElementsIndirect");
+    if (Version >= 460)
+    {
+        pGlMultiDrawArrysIndirectCount = (PFNGLMULTIDRAWARRAYSINDIRECTCOUNTPROC) IRR_OGL_LOAD_EXTENSION("glMultiDrawArraysIndirectCount");
+        pGlMultiDrawElementsIndirectCount = (PFNGLMULTIDRAWELEMENTSINDIRECTCOUNTPROC) IRR_OGL_LOAD_EXTENSION("glMultiDrawElementsIndirectCount");
+    }
+    else if (FeatureAvailable[IRR_ARB_indirect_parameters])
+    {
+        pGlMultiDrawArrysIndirectCount = (PFNGLMULTIDRAWARRAYSINDIRECTCOUNTARBPROC) IRR_OGL_LOAD_EXTENSION("glMultiDrawArraysIndirectCountARB");
```

Comment:
I'd make a `pGlMultiDrawArrayIndirectCountARB` and pick at function call, not exactly safe to assume typedef **PFNGLMULTIDRAWARRAYSINDIRECTCOUNTARBPROC** == **PFNGLMULTIDRAWARRAYSINDIRECTCOUNTPROC**

Reply:
that's why I used cast to *ARB so that gives an error if it's not same. But generally it is